runtime/jitter/jitter.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2023 The Flux authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package jitter
+
+import (
+	"math/rand"
+	"time"
+)
+
+// Duration is a function that takes a duration and returns a modified duration
+// with jitter added.
+type Duration func(time.Duration) time.Duration
+
+// NoJitter returns a Duration function that will return the given duration
+// without modification.
```

Comment:
this seems to suggest that `NoJitter` returns a func, but it just returns the provided `time.Duration` unmodified

Reply:
could be better to do this, to bring it in alignment with `Percent`, but what's here currently is also fine, as long as the comments reflect it properly.
```go
func NoJitter(d time.Duration) Duration {
	return func(d time.Duration) time.Duration {
	   return d
	}
}
```

Reply:
Should have been "is a ...", thanks :bow: